pkg/gui/controllers/helpers/merge_and_rebase_helper.go

```diff
@@ -123,7 +123,17 @@ func (self *MergeAndRebaseHelper) genericMergeCommand(command string) error {
 			self.c.Git().Rebase.GenericMergeOrRebaseActionCmdObj(commandType, command),
 		)
 	}
-	result := self.c.Git().Rebase.GenericMergeOrRebaseAction(commandType, command)
+
+	runCmd := self.c.Git().Rebase.AddSkipEditorCommand(
+		self.c.Git().Rebase.GenericMergeOrRebaseActionCmdObj(commandType, command))
+
+	var result error = nil
+	if self.c.Git().Config.UsingGpg() && command != REBASE_OPTION_ABORT {
+		result = self.c.RunSubprocessAndRefresh(runCmd)
```

Comment:
If you're going to run this in a subprocess, you may as well not skip the editor, right? That is, we should only use AddSkipEditorCommand  in the else block

Reply:
I have just tried your suggestions (latest commit)

It brings up an (external) editor only when using GPG, which is fine. 

But this is a different behavior compared to without GPG signing (else block) where it doesn't bring up any editor at all. Which makes sense since we are skipping the editor I guess?

I am fine either way but shouldn't them be behaving the same (i.e. no editor at all) or is the `SkipEditorCommand` supposed to bring the lazygit editor?